backend/src/main/java/com/github/waymaker7/capstone/BlogsController.java

```diff
@@ -38,6 +38,11 @@ public void deletePost(@PathVariable String id){
         blogsServices.deletePost(id);
     }
 
+    @PutMapping("{id}")
+    public void updatePost(@RequestBody Blogs blogs){
+        blogsServices.updatePost(blogs);
+    }
+
```

Comment:
Als Tipp: behandle eventuell auftretende Fehler direkt. So wie du es in der Methode getBlogs gemacht hast, kannst du dir später Zeit sparen 👍